winui/AutoComplete/Highlighting.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: Highlighting Feature in WinUI AutoComplete control | Syncfusion
+description: Learn here all about Highlighting Feature in Syncfusion WinUI AutoComplete control into WinUI application.
+platform: winui
+control: SfAutoComplete
+documentation: UG
+---
+
+# Highlighting in WinUI AutoComplete(SfAutoComplete)
+
+Highlight matching characters in a suggestion list to pick an item with more clarity. There are two ways to highlight the matching text:
+
+* **Highlight beginning text**
+* **Highlight all occurance of search text**
+
+
+The text highlight can be indicated with various customizing styles by enabling the below properties. They are
+
+* **HighlightedTextColor** - sets the color of the highlighted text for differentiating the highlighted characters.
+
+* **HighlightedTextFontStyle** - sets the font style of the highlighted text.
+        
+* **HighlightedTextFontWeight** - sets the font weight of the highlighted text.
+
+* **HightlightedTextFontSize** - sets the font size of the highlighted text.
+
+{% tabs %}
+{% highlight XAML %}
+    <editors:SfAutoComplete x:Name="autoComplete"
+                            DisplayMemberPath="Name"                       
+                            TextHighlightMode="Matched"                                
+                            TextSearchMode="StartsWith"                               
+                            HighlightedTextFontSize="15"                               
+                            HighlightedTextFontStyle="Italic"                               
+                            HighlightedTextFontWeight="Medium"                                
+                            HighlightedTextForeground="Red"                                
+                            ItemsSource="{Binding SocialMedias}">       
+    </editors:SfAutoComplete>
+
+{% endhighlight %}
+{% highlight C# %}
+        autoComplete.DisplayMemberPath = "Name";
+        autoComplete.TextMemberPath = "Name";
+        autoComplete.TextHighlightMode = AutoCompleteTextHighlightMode.Unmatched;
+        autoComplete.TextSearchMode = AutoCompleteTextSearchMode.Contains;
+        autoComplete.HighlightedTextFontSize = 10;
+        autoComplete.HighlightedTextFontStyle = FontStyle.Normal;
+        autoComplete.HighlightedTextFontWeight = FontWeights.ExtraLight;
+        autoComplete.HighlightedTextForeground = Color.Red;
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete text highlighting properties](Highlighting_images/winui-autocomplete-highlightingproperties.png)
+
+
+
+### **Highlight beginning text**
+It highlights the matches that starts with the typed characters in suggestion list.
```

Comment:
It highlights the matches that start with the typed characters in the suggestion list.